datastore/mongo/organisation_member.go

```diff
@@ -194,8 +200,93 @@ func (o *orgMemberRepo) FetchOrganisationMemberByUserID(ctx context.Context, use
 	member := new(datastore.OrganisationMember)
 	err := o.inner.FindOne(ctx, filter).Decode(member)
 	if errors.Is(err, mongo.ErrNoDocuments) {
-		err = datastore.ErrOrgMemberNotFound
+		return nil, datastore.ErrOrgMemberNotFound
 	}
 
+	err = o.fillOrgMemberUserMetadata(ctx, []*datastore.OrganisationMember{member})
 	return member, err
 }
+
+func (o *orgMemberRepo) fillOrgMemberUserMetadata(ctx context.Context, members []*datastore.OrganisationMember) error {
+	userIDs := make([]string, 0, len(members))
+	for i := range members {
```

Comment:
@danvixent Why can't this `for loop` be one?

Reply:
fixed